src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/LanguageServerTagHelperCompletionServiceTest.cs

```diff
@@ -34,19 +33,19 @@ public void GetAttributeCompletions_OnlyIndexerNamePrefix()
                     .Metadata(PropertyName("RouteValues"))
                     .AsDictionary("asp-route-", typeof(string).FullName))
                 .Build(),
-        };
-        var expectedCompletions = AttributeCompletionResult.Create(new Dictionary<string, HashSet<BoundAttributeDescriptor>>()
+        ];
+        var expectedCompletions = AttributeCompletionResult.Create(new()
         {
-            ["asp-route-..."] = new HashSet<BoundAttributeDescriptor>()
-            {
+            ["asp-route-..."] =
+            [
```

Comment:
Yeah, I totally agree. I have another branch making changes in this file and already made all of these one-liners because they look so weird. I'll make that change here.

Reply:
Fixed